libs/coin-modules/coin-solana/src/__tests__/fixtures/helpers.fixture.ts

```diff
@@ -155,3 +156,25 @@ export function epochInfo({ epoch }: { epoch: number }) {
     transactionCount: 22661093,
   };
 }
+
+export function transaction(raw?: string): Transaction {
+  return {
+    amount: new BigNumber(0),
+    recipient: "",
+    model: {
+      commandDescriptor: {
+        fee: 0,
+        warnings: {},
+        command: {
+          kind: "transfer", // any kind can work, just needed a value for the test
+          sender: "Hj69wRzkrFuf1Nby4yzPEFHdsmQdMoVYjvDKZSLjZFEp",
+          recipient: "DwRL6XkPAtM1bfuySJKZGn2t9WeG25RC39isAu2nwak4",
+          amount: 0,
+        },
+        errors: {},
+      },
+    },
+    raw: raw,
+    family: "solana",
+  } as Transaction;
```

Comment:
Do we need the `as Transaction` here if you provide all the fields ?
You're probably just missing the `kind` in the the model and the empty `uiStates`